app_dart/lib/cocoon_service.dart

```diff
@@ -31,7 +31,7 @@ export 'src/request_handlers/vacuum_github_commits.dart';
 export 'src/request_handling/authentication.dart';
 export 'src/request_handling/body.dart';
 export 'src/request_handling/cache_request_handler.dart';
-export 'src/request_handling/luci_pubsub_authentication.dart';
+export 'src/request_handling/pubsub_authentication.dart';
```

Comment:
Do we want to make this name even more generic (e.g. service_account auth) since its not specifically tied to pubsub service?

Reply:
No, because in the future I want to rename https://github.com/flutter/cocoon/blob/main/app_dart/lib/src/request_handling/swarming_authentication.dart to that